src/components/app-configure/AppKeyspace.vue

```diff
@@ -59,20 +59,22 @@ function handleCancel() {
 <template>
   <section name="app-branding">
     <SettingCard class="app-branding-card">
-      <template #title>Keyspace</template>
+      <template #title>Keyspace Type</template>
       <template #description>
         <div>
-          In the Arcana ecosystem, there are two options for key security: App
-          Specific (default) and Global. The latter option requires extra
-          verification during registration but allows users to use the same keys
-          across all applications. Changing options later will affect user
-          experience as wallet address will change for the user.
+          Choose from app-specific (default, highly secure) or global keyspace
+          for your app. Enabling global keyspace requires an extra verification
+          step during app registration. Global keyspace provides a better user
+          experience as the app users are assigned the same keys (wallet
+          address) across all apps in the Arcana ecosystem. Changing keyspace
+          type after app deployment will affect user experience as the keys are
+          reassigned causing a change in user's wallet address.
```

Comment:
“user's wallet address” → “users' wallet addresses”. Otherwise, add an article.

Reply:
Fair enough.

How about ...

`Changing the keyspace type after app deployment will affect the user experience as the keys are reassigned causing a change in the user's wallet address.`

Reply:
LGTM.